Makefile

```diff
@@ -157,6 +157,22 @@ release-snapshot: install-goreleaser
 	@source $(GO_BUILD_LDFLAGS_CMD) > /dev/null && goreleaser --snapshot --rm-dist
 
 
+.PHONY: docs-container
+docs-container:
+	docker build -t m3db-docs -f hack/Dockerfile-docs docs
+
+.PHONY: docs-build
+docs-build: docs-container
+	docker run -v $(PWD):/m3db --rm m3db-docs "mkdocs build -e docs/theme -t material"
```

Comment:
Does this mean you need to run it from a particular directory? Would it be better to use `$GOPATH`?